codecparsers/h264Parser.cpp

```diff
@@ -82,14 +114,14 @@ static int32_t searchStartPos(const uint8_t* src, uint32_t size)
 }
 
 //7.3.2.1.1.1 Scaling list syntax
-static void scalingList(NalReader& nr, uint8_t* sl, uint32_t size, uint32_t index)
+static bool scalingList(NalReader& br, uint8_t* sl, uint32_t size, uint32_t index)
 {
     uint8_t lastScale = 8;
     uint8_t nextScale = 8;
     int32_t delta_scale;
     for (uint32_t j = 0; j < size; j++) {
         if (nextScale) {
-            delta_scale = nr.readSe();
+            READ_SE(delta_scale);
```

Comment:
Return gets hidden in the macro; making it a inline function will have a return value and hence better readable on this function.